tests/gauge_force_test.cpp

```diff
@@ -1,6 +1,7 @@
 #include <stdio.h>
```

Comment:
Maybe it's time we renamed this test?  Gauge loop test?

Reply:
Maybe gauge path test? Since the forces technically aren't loops. But, yes, I agree we should rename it.

Reply:
Good suggestion

Reply:
https://github.com/lattice/quda/pull/1308/commits/1e3db7ab2f46d71cf3b88aa7e904b8b4f9c8c834